### PR TITLE
Loki: Adjust to logging convention

### PR DESCRIPTION
**Pull request by svennergr**
**What is this feature?**

Based on https://docs.google.com/document/d/1p0wl4WwjD6eeixk3_MCwdqVjW0EucrHDuLKpx4Ge24s/edit and https://github.com/grafana/grafana/pull/74428 logging in Loki needed to be adjusted.

### CODE REVIEW

**Review by matyax — APPROVED** [review]
Looks good!

File: pkg/tsdb/loki/api.go [hunk 1]
```diff
@@ -188,16 +192,16 @@ func (api *LokiAPI) DataQuery(ctx context.Context, query lokiQuery, responseOpts
 
 	iter := jsoniter.Parse(jsoniter.ConfigDefault, resp.Body, 1024)
 	res := converter.ReadPrometheusStyleResult(iter, converter.Options{Dataplane: responseOpts.metricDataplane})
-	took = time.Since(start)
 
 	if res.Error != nil {
 		span.RecordError(res.Error)
 		span.SetStatus(codes.Error, err.Error())
-		logger.Error("Error parsing response from loki", "err", res.Error, "metricDataplane", responseOpts.metricDataplane, "took", took)
+		// TODO: queriesLength, responsesLength
```
[the remark this answers is not in the document]
Comment (ivanahuckova) [reply pkg/tsdb/loki/api.go:199]: This was a note for me to figure out what to do with 1 issue. Fixed.

**Review by svennergr — COMMENTED** [review]
✅ Thanks for taking over, Ivana! LGTM

**Review by matyax — APPROVED** [review]
☄️

**Review by ivanahuckova — APPROVED** [review]

**Pull request merged**